schema_based_on_diagram.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE PATIENTS (
+ID INT GENERATED ALWAYS AS IDENTITY,
+NAME VARCHAR(255) NOT NULL,
+DATE_OF_BIRTH DATE NOT NULL,
+
+PRIMARY KEY(ID)
+);
+
+CREATE TABLE MEDICAL_HISTORIES (
+    ID INT GENERATED ALWAYS AS IDENTITY,
+    ADMITTED_AT TIMESTAMP,
+    PATIENT_ID INT NOT NULL,
+    STATUS VARCHAR(120),
+
+    CONSTRAINT patient_fk
+    FOREIGN KEY (PATIENT_ID)
+    REFERENCES PATIENTS(ID),
+
+    PRIMARY KEY(ID)
+);
+
+CREATE INDEX MEDICAL_INDEX ON MEDICAL_HISTORIES(PATIENT_ID); 
+
+CREATE TABLE INVOICES (
+    ID INT GENERATED ALWAYS AS IDENTITY, 
+    TOTAL_AMOUNT DECIMAL NOT NULL,
+    GENERATED_AT TIMESTAMP,
+    PAYED_AT TIMESTAMP,
+    MEDICAL_HISTORY_ID INT,
+
+    CONSTRAINT medical_history_fk
+    FOREIGN KEY (MEDICAL_HISTORY_ID)
+    REFERENCES MEDICAL_HISTORIES(ID),
+
+    PRIMARY KEY (ID)
+);
+
+CREATE INDEX INVOICES_INDEX ON INVOICES(MEDICAL_HISTORY_ID); 
+
+CREATE TABLE INVOICE_ITEMS (
+    ID INT GENERATED ALWAYS AS IDENTITY,
+    UNIT_PRICE DECIMAL NOT NULL,
+    QUANTITY INT NOT NULL,
+    TOTAL_PRICE DECIMAL NOT NULL,
+    INVOICE_ID INT NOT NULL,
+    TREATMENT_ID INT NOT NULL,
+
+    CONSTRAINT invoice_fk
+    FOREIGN KEY (INVOICE_ID)
+    REFERENCES INVOICES(ID),
+    
+    CONSTRAINT treatment_fk
+    FOREIGN KEY (TREATMENT_ID)
+    REFERENCES TREATMENTS(ID),
+
+    PRIMARY KEY (ID)
+);
+
+CREATE INDEX INVOICE_ITEMS_INDEX ON INVOICE_ITEMS(INVOICE_ID); 
+
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, you missed the index on the `Treatment_id` column. Please include it as, according to the requirements, all foreign keys should have their respective indexes! 🤓 🔧